src/components/lessons/LessonsList.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <div class="bg-white">
+        <div class="mx-auto w-full px-6 py-12 sm:py-16 lg:py-20 lg:px-8">
+            <div class="mx-auto divide-y divide-gray-900/10">
+                <h2 class="text-2xl font-bold leading-10 tracking-tight text-gray-900">Lessons</h2>
+                <dl class="mt-10 space-y-6 divide-y divide-gray-900/10">
```

Comment:
could you explain please, why do you use dl tag here?
also I recommend to recheck this element in html validator
https://validator.w3.org/nu/#textarea

Reply:
These are also features of using tailwind